docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #165.        PL = np.where(PL==np.inf, np.NINF, PL)
</pre><p> </p><p>Replace it with:</p><pre class="ql-syntax" spellcheck="false">mask = PL == np.inf
PL[mask] = np.NINF
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
The code is chagned to return the original PL as well. Hence, we keep `PL` and assign its modified version to a new variable.

```
PL_modified = np.where(PL==np.inf, np.NINF, PL)
```